web-components/src/components/team-section/index.tsx

```diff
@@ -62,10 +62,9 @@ const TeamSection = ({
   children,
 }: TeamSectionProps): JSX.Element => {
   const classes = useStyles();
-  const sorted = members.sort((a, b) =>
-    firstName(a.name) > firstName(b.name) ? 1 : -1,
-  );
-  const teamMembers = alphabetized ? sorted : members;
+  const sortedMembers = alphabetized // avoid sorting if not alphabetized
+    ? members.sort((a, b) => (firstName(a.name) > firstName(b.name) ? 1 : -1))
+    : members;
```

Comment:
Left this in just in case we want to use in future, but changed the conditional so it won't calculate needlessly